micropip/package_manager.py

```diff
@@ -6,31 +6,52 @@
 from micropip import package_index
 from micropip._commands import mock_package
 from micropip.freeze import freeze_lockfile
+from micropip.install import install
 from micropip.list import list_installed_packages
 from micropip.package import PackageDict
```

Comment:
Let's change import paths to relative paths.

```
from .package import PackageDict
# same for other imports
```

Reply:
Sure, but I'm curious about the reasoning behind this change? (Personally I don't see a significant advantage over one another)

Reply:
It distinguishes the local imports from external imports. Also, a minor benefit would be that if we change the package name, it does not need to be updated.